config/routes.rb

```diff
@@ -1,5 +1,11 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: 'pages#home'
+  resources :users, only: [ :new, :create, :show ] do
```

Comment:
ich würde da nochmal checken, ob sich das irgendwie mit den devise routes in die quere kommt die ja darüber schon definiert werden